Formula/slackcat.rb

```diff
@@ -13,38 +11,17 @@ class Slackcat < Formula
     sha256 "b97794811a471a2cd1cb0ef9cd0045b2e2b68502c7c979239eda0b1ec56c9db3" => :el_capitan
   end
 
+  depends_on "dep" => :build
   depends_on "go" => :build
 
-  go_resource "github.com/BurntSushi/toml" do
-    url "https://github.com/BurntSushi/toml.git",
-        :revision => "a368813c5e648fee92e5f6c30e3944ff9d5e8895"
-  end
-
-  go_resource "github.com/bluele/slack" do
-    url "https://github.com/bluele/slack.git",
-        :revision => "307046097ee9f07094bb547c5d96d86d759054a6"
-  end
-
-  go_resource "github.com/fatih/color" do
-    url "https://github.com/fatih/color.git",
-        :revision => "5df930a27be2502f99b292b7cc09ebad4d0891f4"
-  end
-
-  go_resource "github.com/skratchdot/open-golang" do
-    url "https://github.com/skratchdot/open-golang.git",
-        :revision => "75fb7ed4208cf72d323d7d02fd1a5964a7a9073c"
-  end
-
-  go_resource "github.com/urfave/cli" do
-    url "https://github.com/urfave/cli.git",
-        :revision => "119bb6564841921ce6f1401e0f5d75317bdd9f4d"
-  end
-
   def install
     ENV["GOPATH"] = buildpath
-    Language::Go.stage_deps resources, buildpath/"src"
-    system "go", "build", "-o", bin/"slackcat",
+    (buildpath/"src/github.com/vektorlab/slackcat").install buildpath.children
+    cd "src/github.com/vektorlab/slackcat" do
```

Comment:
Using this style of installation you'll likely need a `prefix.install_metafiles` as the last line of this block.

Reply:
Thats twice now I've forgotten about `metafiles`. 😞  

Thanks @DomT4!

Reply:
No worries! It happens so often I have a GitHub quick reply for it _(not used here)_ so if you only slip twice handling quite a few of these you're not doing too badly at all 😸.